spec/message.abnf

```diff
@@ -55,7 +55,10 @@ name-start = ALPHA / "_"
            / %x2070-218F / %x2C00-2FEF / %x3001-D7FF
            / %xF900-FDCF / %xFDF0-FFFD / %x10000-EFFFF
 name-char = name-start / DIGIT / "-" / "." / ":"
-          / %xB7 / %x0300-036F / %x203F-2040
+          / %xB7 / %x300-36F / %x203F-2040
+unquoted = unquoted-start *name-char
```

Comment:
nit: insert a blank link before `unquoted` to separate the new production from the ones about `name` and `nmtoken` above (or, potentially, move the `unquoted` production with `name` and `nmtoken`???)

Reply:
```suggestion

unquoted = unquoted-start *name-char
```